src/main/webapp/app/tasks/task-search/task-search.component.ts

```diff
@@ -0,0 +1,26 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+
+/**
+ * Search field for tiny tasks.
+ */
+@Component({
+  selector: 'tiny-task-search',
+  templateUrl: './task-search.component.html',
+  styleUrls: ['./task-search.component.scss']
+})
+export class TaskSearchComponent {
+
+  taskSearch: FormGroup = new FormGroup({
+    queryInput: new FormControl('', Validators.required)
+  });
```

Comment:
I don't see the need to have a validator here, it looks more like a unnecessary copy&paste form the task-form.component.ts - adding also to this conclusion is the fact, that the validation does not even work are the is no `formControlName` with the name `queryInput`